src/wp-includes/option.php

```diff
@@ -253,6 +253,88 @@ function get_option( $option, $default_value = false ) {
 	return apply_filters( "option_{$option}", maybe_unserialize( $value ), $option );
 }
 
+/**
+ * Primes specific options into the cache with a single database query.
+ *
+ * Only options that do not already exist in cache will be primed.
+ *
+ * @since 6.4.0
+ *
+ * @global wpdb $wpdb WordPress database abstraction object.
+ *
+ * @param array $options An array of option names to be primed.
+ */
+function prime_options( $options ) {
+	$alloptions     = wp_load_alloptions();
+	$cached_options = wp_cache_get_multiple( $options, 'options' );
+
+	// Filter options that are not in the cache.
+	$options_to_prime = array();
+	foreach ( $options as $option ) {
+		if ( ! isset( $cached_options[ $option ] ) && ! isset( $alloptions[ $option ] ) ) {
```

Comment:
The `! isset( $cached_options[ $option ] )` is problematic since that would only work if `$cached_options[ $option ]` was `null` for options that aren't in cache. But `wp_cache_get()` and `wp_cache_get_multiple()` return `false` for options that aren't found.
```suggestion
		if ( ( ! isset( $cached_options[ $option ] ) || ! $cached_options[ $option ] ) && ! isset( $alloptions[ $option ] ) ) {
```